articles/azure-resource-manager/management/delete-resource-group.md

```diff
@@ -13,6 +13,9 @@ ai-usage: ai-assisted
 
 This article shows how to delete resource groups and resources. It describes how Azure Resource Manager orders the deletion of resources when you delete a resource group.
 
+> [!NOTE]  
+> To delete a resource group, firstly you need to delete its underlying resource locks as well as backup data.
```

Comment:
Can you add a link to the resource locks?  For example:

To delete a resource group, you must first remove any underlying [resource locks](./lock-resources.md) and backup data.

Reply:
Looks good. Let me try to change this.